mod-audit-server/src/main/java/org/folio/util/inventory/InventoryEventType.java

```diff
@@ -0,0 +1,5 @@
+package org.folio.util.inventory;
+
+public enum InventoryEventType {
+  UPDATE, DELETE, CREATE, DELETE_ALL, REINDEX, ITERATE, MIGRATION
```

Comment:
I would suggest to replace DELETE_ALL, REINDEX, ITERATE, MIGRATION with UNKNOWN and create a method with `@JsonCreator` annotation for deserialization from a string to handle mapping anything that is not in UPDATE, DELETE, CREATE list to UNKNOWN .

Reply:
Same in `InventoryDeleteEventSubType`. This will make us safer in case of any changes in this types.